stored_procedures/README.md

```diff
@@ -11,6 +11,30 @@ you can reference it like the following:
 DECLARE next_ids ARRAY<INT64> DEFAULT [];
 CALL bqutil.procedure.GetNextIds(10, next_ids);
 ```
+## Using the stored procedures
+
+All stored procedures within this repository are available under the `bqutil` project on
+publicly shared datasets. Queries can then reference the shared procedures in the US multi-region via
+`bqutil.<dataset>.<procedure>()`.
```

Comment:
We already know the dataset name is procedure so just add placeholder for the procedure name. 
E.g. bqutil.procedure.<procedure_name>()

Reply:
Done